contents/tutorials/session-metrics.md

```diff
@@ -0,0 +1,87 @@
+
+![Session duration sum](../images/tutorials/session-metrics/session-duration-sum.png)
+
+Once you modify the graph to your liking, by choosing a date range and setting the Y-axis unit to `Duration (s)`, you have a nice graph of the sum of time on site over time.
+
+![Session duration sum graph](../images/tutorials/session-metrics/session-duration-sum-graph.png)
+
+This might be useful for getting a rough idea of the usage of our whole product over time but becomes more useful as you break it down. You can do this by filtering the data. For example, you can filter for sessions by section of your product or site such as pricing or signup.
+
+## Time in sections
+
+To visualize time in sections, add a filter where the `Current URL` contains the section URL you want. In this case, it will be pricing and signup. Once one is set up, use the copy button and then change the `Current URL` value. This makes it quick to add multiple sections to the graph.
+
+> **Tip:** Filter out pages you don’t want to be included, such as your main application on queries related to you marketing site.
+
+![Session duration filters](../images/tutorials/session-metrics/session-duration-filters.png)
+
+Once done, you’ll have a graph of session duration for the signup and pricing sections. Looks like people are spending a lot more time on pricing pages than signup, but remember that this is the *sum of session duration*, averages will be different.
+
+![Pricing vs signup session durations](../images/tutorials/session-metrics/pricing-vs-signup.png)
+
+## Time on pages
+
+If you want to know exactly what pages people are spending the most time in, you can *breakdown by* `Current URL` . Resetting the filters and switching the chart type to `Value` gives you a visual that looks like this:
+
+![Current URL values](../images/tutorials/session-metrics/current-url-values.png)
+
+These insights and visualization might help you reprioritize pages and sections people are spending more time on. Depending on the context, high time on a page could mean the page is important, confusing, or not optimized. [Session recordings](https://posthog.com/product/session-recording) can help you find out if you don’t know.
+
+## Average session duration
+
+Now that you visualized time on site, time in sections, and time on page, we can visualize some session-based metrics based on averages. Averages provide better insight into how individual users spend their time compared to the sum of time spent metrics calculated and visualized in the last section.
+
+We’ll start with the average session duration. It can be calculated by again choosing pageview, but now aggregating by average session duration. Ensuring Y-axis is set to `Duration (s)` creates a graph of average session duration. 
+
+![Average session duration](../images/tutorials/session-metrics/average-session-duration.png)
```

Comment:
Could you edit the image to highlight the Y-axis box in some way? Will help people locate it.